utils/action.js

```diff
@@ -0,0 +1,19 @@
+const Action = require('../models/Action');
+
+/**
+ * helper for logging a user action to the `action` table
+ *
+ * @param {object} user
```

Comment:
Does it need the complete user object? Maybe we can only pass the user id since that is what ends up in the DB?

Reply:
just tested an it works with both. you can pass the user id or the user object. will update the documentation

Reply:
Interesting. Thanks for checking!